### PR TITLE
Add more homepage header style

### PR DESCRIPTION
**Pull request by laurawhalin**

### CODE REVIEW

File: app/assets/stylesheets/layout.css.scss [hunk 1]
```diff
@@ -32,56 +32,62 @@ a {
...
+  margin-right: -30px;
+}
+
+.logout {
+  margin-top: 2px;
+}
+
+.search-nav {
+  background-color: #D7D8D9;
+  margin: 0;
```
Comment (rshea303) [inline app/assets/stylesheets/layout.css.scss:66]: if margin is 0 do we need to declare it?

File: app/assets/stylesheets/layout.css.scss [hunk 2]
```diff
@@ -32,56 +32,62 @@ a {
...
-  margin-top: 10px;
-  padding-left: 69px;
+  margin-left: -21px;
 }
 
-#user-nav {
-  text-align: right;
+.btn-default.view-items {
+  height: 35px;
+  padding-top: 7px;
```
Comment (rshea303) [inline app/assets/stylesheets/layout.css.scss:76]: have you considered a padding-top of 6px or even 8px?  Have you really considered all the alternatives to 7px?

File: app/assets/stylesheets/layout.css.scss [hunk 3]
```diff
@@ -92,14 +98,25 @@ a {
...
-#cart {
-  float: right;
-  margin-top: 10px;
-  padding-right: 40px;
+.container {
+  width: 100%;
+  padding-right: 0px;
+  padding-left: 0px;
+  background-color: white;
+  // margin-top: 170px;
```
Comment (rshea303) [inline app/assets/stylesheets/layout.css.scss]: lots of commented-out css that needs to be removed.  This is the last time I am going to warn you before firing you from the project.

File: app/assets/stylesheets/layout.css.scss [hunk 4]
```diff
@@ -92,14 +98,25 @@ a {
...
+  // z-index: 2;
+  // padding-bottom: 150px;
 }
 
-.logout {
-  margin-top: 2px;
+.alert-danger {
```
Comment (rshea303) [inline app/assets/stylesheets/layout.css.scss:120]: The only 'alert-danger' is see on the horizon is more commented-out code!!!

File: app/assets/stylesheets/layout.css.scss [hunk 5]
```diff
@@ -120,17 +136,14 @@ a {
   padding: 475px 0 0 50px;
 }
 
+//
```
Comment (rshea303) [inline app/assets/stylesheets/layout.css.scss]: Remove me.

File: app/assets/stylesheets/layout.css.scss (hunk 1 above)
Comment (laurawhalin) [inline app/assets/stylesheets/layout.css.scss:66]: This element was inheriting a margin that I wanted to zero out.

**Comment by rshea303** [comment]
Excellent redesign of the header!

**Comment by rshea303** [comment]
Please comment out the two failing tests that are affecting the build.

**Pull request merged**